### PR TITLE
Do not make suggestions for related titles on edit pages

### PR DESCRIPTION
**Pull request by taitus**
## References
Related Issue: #2980 

## Objectives
Disable suggest title feature when editing a resource (Proposal, Debates and Investments)

### CODE REVIEW

**Review by Senen — APPROVED** [review]

File: spec/system/budgets/investments_spec.rb [hunk 1]
```diff
@@ -661,6 +661,16 @@ def investments_order
           expect(page).not_to have_content "You are seeing"
         end
       end
+
+      scenario "Don't show suggestions for edit action", :js do
```
Comment (javierm) [inline spec/system/budgets/investments_spec.rb:665]: `:js` is not needed anymore :tada:.

File: spec/system/debates_spec.rb [hunk 2]
```diff
@@ -770,6 +770,17 @@
         expect(page).not_to have_content "You are seeing"
       end
     end
+
+    scenario "Don't show suggestions for edit action", :js do
```
Comment (javierm) [inline spec/system/debates_spec.rb:774]: `:js` is not needed anymore :tada:.

File: spec/system/proposals_spec.rb [hunk 3]
```diff
@@ -1440,6 +1440,17 @@
         expect(page).not_to have_content "You are seeing"
       end
     end
+
+    scenario "Don't show suggestions for edit action", :js do
```
Comment (javierm) [inline spec/system/proposals_spec.rb:1444]: `:js` is not needed anymore :tada:.

File: spec/system/budgets/investments_spec.rb [hunk 4]
```diff
@@ -661,6 +661,16 @@ def investments_order
...
+        investment = create(factory, title: "Title, has search term", budget: budget, author: author)
+
+        login_as(author)
+        visit edit_budget_investment_path(budget, investment)
+        fill_in "Title", with: "search", fill_options: { clear: :backspace }
```
Comment (javierm) [inline spec/system/budgets/investments_spec.rb:670]: `clear: :backspace` won't be needed after pull request #4456. Shall we rebase this pull request so it depends on #4456?

**Review by javierm — APPROVED** [review]

**Pull request merged**